### ISSUE TITLE
Highlighted menu font colour doesn't respect high contrast theme settings in Windows

### ISSUE THREAD

**Issue by tomasz1986** [keepassxreboot/keepassxc#7119]
The font colour used in the menu doesn't seem to respect the current high contrast theme highlighted text colour settings.

For instance, when using the built-in High Contrast White theme, the highlighted font colour is supposed to be white, however KeePassXC seems to be hardcoded to use black, making the highlighted text basically unreadable.

![image](https://user-images.githubusercontent.com/5626656/141278743-2b2cd064-be2d-406a-a37d-93ad22644db5.png)

You can reproduce the issue easily by turning on the High Contrast White theme, and then launching KeePassXC and hovering over the menu entries.

**Edit**: I forgot to mention the screenshot comes from Windows 10, but I wouldn't be surprised if all Windows versions are affected. Also, even though I said "menu" in the title, it really affects everything in the GUI, e.g. selected folders and entries also always have their font colour set to black.

Windows high contrast themes use separate values for the highlight colour and the highlighted text colour. The two are represented as follows.

```
[Control Panel\Colors]
Hilight=10 36 106
HilightText=255 255 255
```

KeePassXC seems to honour only the first one.

**Comment by droidmonkey** [comment]
Looks like classic theme which makes this a qt problem.
